spec/lib/topics_filter_spec.rb

```diff
@@ -104,6 +139,59 @@
         end
       end
 
+      describe "new / unread operators" do
+        fab!(:user_for_new_filters) { Fabricate(:user) }
+        let!(:new_topic) { Fabricate(:topic) }
+        let!(:unread_topic) do
+          Fabricate(:topic, created_at: 2.days.ago).tap do |t|
+            _post1 = Fabricate(:post, topic: t)
+            _post2 = Fabricate(:post, topic: t)
```

Comment:
```suggestion
            Fabricate(:post, topic: t)
            Fabricate(:post, topic: t)
```

removing unused assignments